monitoring/metrics.html.md

```diff
@@ -416,21 +419,24 @@ This kind of token or "[macaroon](https://fly.io/blog/macaroons-escalated-quickl
 Create an org-restricted token:
 
 ```
-fly token create org -o THE_ORGANIZATION
+fly tokens create org -o THE_ORGANIZATION
 ```
 
 Create a read-only org-restricted token:
 
 ```
-fly token create readonly
+fly tokens create readonly
 ```
 
 These tokens look like this once generated: `FlyV1 fm2_lJPECAAAAAAAAC7txBAzYI6PRWhHLT...(a lot of base64-encoded text)`.
 
 #### Using tokens
-To use one of these tokens in an HTTP header, the "FlyV1" identifier replaces the "Bearer" token identifier. So it would look like this:
+Use the correct scheme based on token type:
 
+- `Bearer <token>` → for `flyctl auth token`.
+- `FlyV1 <token>` → for all tokens created with `fly tokens create`.
 
+Example with a token created via `fly tokens create readonly`:
```

Comment:
The other mention in L437 of `fly tokens create` doesn't mention `readonly` -- it should probably be consistent (unless it's required in your example, in that case, you could modify L437 to list both (`fly tokens create` and `fly tokens create readonly`